src/app/pages/PlantPage/PlantPage.tsx

```diff
@@ -52,7 +50,7 @@ function PlantPage(): JSX.Element {
               variant="newPlantName"
               placeholder="Please enter a name"
               value={plant.name}
-              onNameChange={console.log}
+              onChange={console.log}
```

Comment:
is the 'console.log' a placeholder? if so, do you need the onChance here?

Reply:
The problem is I can't delete it because it is a given prop from the component.